### PR TITLE
Update laptop script to handle both arm and intel architectures

### PR DESCRIPTION
**Pull request by LouisBoudreau**
# Description

The goals of this PR are:

1. Handle both arm and intel architectures
2. Enable this script to be re-runned successfully so that we can use this script to update our dependencies instead of running manual commands every time we update a lib (ex: Node or PSQL)

### CODE REVIEW

**Review by ericmatte — COMMENTED** [review]
Looks good. I tried running it on my already-configured setup (intel) and it seems ok.

I did get that message and after that the script exited with code 0; not sure if this is expected or not 🤔 

image


@LouisBoudreau, does this PR also closes  #4 ?

File: mac [hunk 1]
```diff
@@ -4,13 +4,42 @@
 # Be prepared to turn your laptop (or desktop, no haters here)
 # into an awesome development machine.
 
+show_help() {
```
Comment (ericmatte) [inline mac:7]: After running the script on my machine, now I have this error with foreman 😞 

image

File: mac [hunk 2]
```diff
@@ -78,107 +134,139 @@ case "$SHELL" in
...
+    brew "the_silver_searcher"
+    brew "tmux"
+    brew "vim"
+    brew "zsh"
+    brew "jq"
+    brew "bind"
+
+    # GitHub
+    brew "hub"
+    brew "gh"
```
Comment (charlesvallieres) [inline mac:177]:
```suggestion
```
Same question here, I doubt anyone uses gh or hub

File: mac [hunk 3]
```diff
@@ -78,107 +134,139 @@ case "$SHELL" in
...
+  brew bundle --file=- <<EOF
+    tap "homebrew/services"
+    tap "puma/puma"
+    tap "hashicorp/tap"
+
+    # Unix
+    brew "git"
+    brew "openssl@1.1"
+    brew "the_silver_searcher"
+    brew "tmux"
```
Comment (charlesvallieres) [inline mac:169]:
```suggestion
```
Nobody @ Agendrix uses tmux I think

**Pull request merged**